common/python/error_code/error_status.py

```diff
@@ -102,3 +102,9 @@ class WPERegisterStatus(IntEnum):
 class WOPreprocessStatus(IntEnum):
     SUCCESS = 0
     FAILURE = 1
+
+
+@unique
+class WOParamLength(IntEnum):
```

Comment:
We should ideally be creating a separate file for Enum/constant and only keep error code here strictly. But for the time being let this be here.

Reply:
This is not a error code and I would suggest to keep this enum outside this file